load-balancer/response-time/src/test/java/io/smallrye/stork/loadbalancer/leastresponsetime/LeastResponseTimeLoadBalancerTest.java

```diff
@@ -113,19 +106,6 @@ void shouldSelectFastest() {
 
     }
 
-    @Test
-    void shouldThrowNoServiceInstanceOnNoInstances() throws ExecutionException, InterruptedException {
```

Comment:
We would need a way to re-implement such a test.

Reply:
done